### PR TITLE
Update Caption Text

### PR DESCRIPTION
**Pull request by bryan-umaguing-wedbush**
[ProjID] Changed caption text

### CODE REVIEW

File: _slides/05-bryan-umaguing-wedbush.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Test caption 4356
```
Comment (divyasamuel-wedbush) [inline _slides/05-bryan-umaguing-wedbush.md:6]: wow...

Comment (divyasamuel-wedbush) [inline _slides/05-bryan-umaguing-wedbush.md:6]: Testing start

Comment (hershkashyap) [inline _slides/05-bryan-umaguing-wedbush.md:6]: '''sugestion
looks gooooood
'''

Comment (divyasamuel-wedbush) [inline _slides/05-bryan-umaguing-wedbush.md:6]:
```suggestion
Test caption #4356
```

**Pull request merged**